code/game/objects/effects/anomalies/anomalies_hallucination.dm

```diff
@@ -18,6 +19,8 @@
 	var/turf/open/our_turf = get_turf(src)
 	if(istype(our_turf))
 		hallucination_pulse(our_turf, 5)
+	pixel_x = pixel_x + clamp(rand(-5, 5), -16, 16)
```

Comment:
what's this for?

Reply:
makes the anomaly shift around (like the desc implies)